rsqueakvm/primitives/arithmetic.py

```diff
@@ -285,7 +285,7 @@ def func(interp, s_frame, f):
 
 @expose_primitive(MAKE_POINT, unwrap_spec=[int, int])
 def func(interp, s_frame, x, y):
-    w_res = interp.space.w_Point.as_class_get_shadow(interp.space).new(2)
+    w_res = interp.space.w_Point.as_class_get_shadow(interp.space).new()
```

Comment:
That argument is "extrasize". Points don't need two extra variable fields after the fixed ones dictated by the class format...